[2025-09-17T11:02:36.143+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-17T11:02:36.179+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_verify.task_3 scheduled__2025-09-16T00:00:00+00:00 [queued]>
[2025-09-17T11:02:36.198+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_verify.task_3 scheduled__2025-09-16T00:00:00+00:00 [queued]>
[2025-09-17T11:02:36.199+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-09-17T11:02:36.230+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): task_3> on 2025-09-16 00:00:00+00:00
[2025-09-17T11:02:36.244+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_verify', 'task_3', 'scheduled__2025-09-16T00:00:00+00:00', '--job-id', '168', '--raw', '--subdir', 'DAGS_FOLDER/dag_verify.py', '--cfg-path', '/tmp/tmpafluoesw']
[2025-09-17T11:02:36.253+0000] {standard_task_runner.py:105} INFO - Job 168: Subtask task_3
[2025-09-17T11:02:36.256+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=209) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-17T11:02:36.257+0000] {standard_task_runner.py:72} INFO - Started process 219 to run task
[2025-09-17T11:02:36.358+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_verify.task_3 scheduled__2025-09-16T00:00:00+00:00 [running]> on host 4461bcf72552
[2025-09-17T11:02:36.528+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_verify' AIRFLOW_CTX_TASK_ID='task_3' AIRFLOW_CTX_EXECUTION_DATE='2025-09-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-16T00:00:00+00:00'
[2025-09-17T11:02:36.530+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-17T11:02:37.023+0000] {logging_mixin.py:190} INFO - Matplotlib version: 3.9.2
[2025-09-17T11:02:37.024+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-09-17T11:02:37.042+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-17T11:02:37.043+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_verify, task_id=task_3, run_id=scheduled__2025-09-16T00:00:00+00:00, execution_date=20250916T000000, start_date=20250917T110236, end_date=20250917T110237
[2025-09-17T11:02:37.120+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-17T11:02:37.150+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-17T11:02:37.181+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
